### PR TITLE
[CHORE] Updating Plaza Server List

### PR DESCRIPTION
**Pull request by 0xSacul**
# Description

This PR brings a little update to the server list and rename Oasis to Bumpkin Bazaar which will become the trading server.
(Only the server name has been changes, id is still `sunflorea_oasis`)

This way we separate people wanting to trade and those wanting to chill and chat.

Screenshot 2023-08-29 at 02 23 51
Screenshot 2023-08-29 at 02 24 10

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

local testing

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/world/mmoMachine.ts [hunk 1]
```diff
@@ -32,18 +32,46 @@ export type ServerId =
...
 
-type ServerName = "Bliss" | "Dream" | "Oasis" | "Brazil" | "Magic";
+type ServerName =
+  | "Bliss"
+  | "Dream"
+  | "Oasis"
+  | "Brazil"
+  | "Magic"
+  | "Bumpkin Bazaar";
+type ServerPurposes = "Chill & Chat" | "Trading";
```
Comment (spencerdezartsmith) [inline src/features/world/mmoMachine.ts:42]: Small thing, the type should be singular ie `ServerPurpose` as it will only ever be one.

**Review by spencerdezartsmith — COMMENTED** [review]
Small change re. type name

File: src/features/world/mmoMachine.ts (hunk 1 above)
Comment (0xSacul) [reply src/features/world/mmoMachine.ts]: Ooops, nice find thank you ❤️

**Review by spencerdezartsmith — APPROVED** [review]
LGTM

**Pull request merged**